Fix mentioning Slack users/groups (#21734)
grafana/grafana: CHANGELOG.md

# 6.7.0 (unreleased)

## Breaking changes

> * **Slack**: Remove Mention setting and instead introduce _Mention Users_, _Mention Groups_, and _Mention Channel_. The first two settings require user and group IDs respectively. This change was necessary due to the fact that the way of mentioning via the Slack API changed, and mentions in Slack notifications no longer worked.

Reviewer: You could add a link mentioning the modification in the Slack API.

Author: @papagian Done.

> * **Slack**: Remove Mention setting and instead introduce _Mention Users_, _Mention Groups_, and _Mention Channel_. The first two settings require user and group IDs respectively. This change was necessary due to the fact that the way of mentioning via the Slack API [changed](https://api.slack.com/changelog/2017-09-the-one-about-usernames) and mentions in Slack notifications no longer worked.

Reviewer 2: ```suggestion
* **Slack**: Removed _Mention_ setting and instead introduce _Mention Users_, _Mention Groups_, and _Mention Channel_. The first two settings require user and group IDs, respectively. This change was necessary because the way of mentioning via the Slack API [changed](https://api.slack.com/changelog/2017-09-the-one-about-usernames) and mentions in Slack notifications no longer worked.
```
